CONTRIBUTING.md

```diff
@@ -0,0 +1,185 @@
+- **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+- **Explain which behavior you expected to see instead and why.**
+- **Include screenshots and animated GIFs** if necessary.
+- **If the problem is related to performance or memory**, include a [CPU profile capture](https://developers.google.com/web/tools/chrome-devtools/rendering-tools/js-execution) with your report.
+- **If the problem only occurs in a specific browser** describe the version and name of the browser.
+
+Provide more context by answering this question:
+
+- **Can you reliably reproduce the issue?** If not, provide details about how often the problem happens and under which conditions it normally happens.
+
+### Your First Pull Request
+
+If you want to contribute to this project and do not know where to start [good first issues](https://github.com/liferay/clay/labels/good%20first%20issue) is a great place to start.
+
+If you decide to fix an issue, please be sure to check the comment thread in case somebody is already working on a fix. If nobody is working on it at the moment, please leave a comment stating that you intend to work on it so other people don’t accidentally duplicate your effort.
+
+### Proposing a Change
+
+This section guides you through how to propose a public API change, make non-trivial changes to the implementation, or propose something new. Following these guidelines helps maintainers understand your suggestion and your point of view.
+
+#### Before submitting a suggestion for improvement
+
+- **Perform a cursory search** to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
+
+#### How do I submit a suggestion for improvement?
+
+Suggestions for enhancement or API changes are tracked as [Github issues](https://github.com/liferay/clay/issues). Please create a problem in this repository and provide the following information:
+
+- **Use a clear and descriptive title** for the issue to identify the suggestion.
+- **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+- **Describe the current behavior** and explain which behavior you expected to see instead and why.
+- **Provide details of the impact your change will cause** and how you will proceed with it.
+- **Provide the pros and cons of your change.**
+- **Provide at least one alternative for your change.**
+
+### Sending a Pull Request
+
+This section guides you how to proceed with sending a pull requests. The main staff is always monitoring pull requests. Following these guidelines helps the maintainers to do thorough review.
+
+> Since travis is failing on executing **a11y tests** due to out of date suid you need to execute `npm run pa11y` locally before sending any pr to verify we're following accessibility standars.
```

Comment:
```suggestion
> Since Travis is failing on executing **a11y tests** due to out of date suid you need to execute `npm run pa11y` locally before sending any PR to verify we're following accessibility standards.
```